src/components/views/dialogs/SpotlightDialog.tsx

```diff
@@ -596,9 +595,9 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", onFinished }) =>
     }
 
     const onDialogKeyDown = (ev: KeyboardEvent) => {
-        const navAction = getKeyBindingsManager().getNavigationAction(ev);
+        const navAction = getKeyBindingsManager().getAccessibilityAction(ev);
         switch (navAction) {
-            case "KeyBinding.closeDialogOrContextMenu" as KeyBindingAction:
+            case KeyBindingAction.Escape:
             case KeyBindingAction.FilterRooms:
                 ev.stopPropagation();
                 ev.preventDefault();
```

Comment:
This is confusing Navigation and Accessibility actions and FilterRooms won't work anymore.

Can we fix-up the typings so that Typescript catches this scenario by screaming at us for saying that `KeyBindingAction.FilterRooms` can never be returned from `getNavigationAction`?